internal/commands/app/init_inputs.go

```diff
@@ -14,7 +14,7 @@ const (
 	flagProjectUsage = "the MongoDB cloud project id"
 
 	flagFrom      = "from"
-	flagFromShort = "s"
+	flagFromShort = "a"
```

Comment:
[minor thought] I am wondering if, once the commands are implemented, we want to take a look at all of the flags and their shorthands to see if anything could be more intuitive here

Reply:
On that same thought, with a flag as short as `--from` it might not necessarily warrant a shorthand, unless it's the most common used flag (in which case it probably should get prioritized to get the `-f`). This thought goes towards our flags that are global or across multiple commands, unless they are to be used often removing their shorthand might make more sense if there is flag that is going to be used more even if it is local to a single command.

Reply:
yeah, in general polish is needed across the board for command flags and usages.  for now, we'll stick with whatever works, and get product/design to chime in on the final say once we've got more of the command suite built out (and we can really see what our common options are, etc)